pkg/pod/status.go

```diff
@@ -266,11 +279,30 @@ func createMessageFromResults(results []result.RunResult) (string, error) {
 	return string(bytes), nil
 }
 
+func findStepResultsFetchedByTask(stepName string, specResults []v1.TaskResult) (map[string]string, error) {
```

Comment:
do you mind adding docstrings for this function, and example input and output?

Reply:
done.